app/views/idv/in_person/ssn.html.erb

```diff
@@ -1,15 +1,5 @@
-<% title t('titles.doc_auth.verify') %>
-
-<h1 class="h3 margin-y-0">
-  <%= t('in_person_proofing.headings.ssn') %>
-</h1>
-
-<%= validated_form_for :doc_auth,
-                       url: url_for,
-                       method: 'put',
-                       html: { autocomplete: 'off', class: 'margin-y-5' } do |f| %>
-  <%= f.button :button,
-               t('doc_auth.buttons.continue'),
-               type: :submit,
-               class: 'usa-button--big usa-button--wide' %>
+<% if applicant_ssn %>
+  <%= render 'idv/doc_auth/ssn_update', flow_session: flow_session %>
+<% else %>
+  <%= render 'idv/doc_auth/ssn_init', flow_session: flow_session, should_render_id_success_alert: false %>
```

Comment:
Now that we're using the ssn page across doc auth and IPP I think it's worth rolling it up into somewhere like views/idv/shared. I'll do this in a separate PR

Reply:
I'd also love if we could combine `ssn_update` and `ssn_init`, and parameterize the remaining variables in the same way you did with the success alert here (i.e. heading text, footer link).

Reply:
I think I can probably throw that in! If it gets complicated I'll spin up a ticket for it